phpunit.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="./vendor/phpunit/phpunit/phpunit.xsd"
+         bootstrap="vendor/autoload.php"
+         colors="true"
+>
+    <testsuites>
+        <testsuite name="Unit">
+            <directory suffix="Test.php">./tests/Unit</directory>
+        </testsuite>
+    </testsuites>
+</phpunit>
```

Comment:
I think this file isn't necessary.

Reply:
Actually, this file is necessary, if we want to add default configurations to PHPUnit, so that we don't have to add lengthy PHPUnit command with configuration arguments. 

If you think it is not needed now, we can remove it as well but at someday we will need it. 

@note file name should be `phpunit.xml.dist` and we can add phpunit.xml into gitignore .


I hope that clarifies the things. Let me know if there is anything ..